layouts/DefaultLayout.tsx

```diff
@@ -47,24 +46,16 @@ const DefaultLayout: FC<PropsWithChildren> = ({ children }) => {
 
   return (
     <div
-      className={`relative min-h-screen justify-evenly overflow-x-hidden ${
+      className={`${inter.variable} ${kanit.variable} ${
```

Comment:
I would probably put this on the body element in _document. Should fix the issue @Robiquet reported